### DISCUSSION TITLE
How to map path temporary closed

### DISCUSSION THREAD

**Discussion by drazde** [osmandapp/OsmAnd#12720] [Maps Related]
Hi, 
I'm a mapper and I use OSMAnd for hiking.
Many times I arrived on path that was closed for landslide or work; how can I map this?

Is it possible view (and used this info for routing) on OSMAnd a path is closed temporary?

This feature would be particularly useful for who do hiking or mtb

**Comment by vshcherb** [comment]
We recommend to use conditional tags. Please keep mind that you can conditionally restrict access in 2 ways: ``` access= no + access:conditional=yes @ future data to open )... ``` or ``` access=yes + access:conditional=no @ date now - future data to open)... ```

**Reply by tommycrock** [reply]
It would be worth noting that if there are any more specific tags than access, such as foot, they would (or should) override access, so the conditional restrictions would need to be placed there. Also worth noting that very temporary changes are normally thought to not be good practice https://wiki.openstreetmap.org/wiki/Good_practice#Don.27t_map_temporary_events_and_temporary_features

**Comment by drazde** [comment]
@vshcherb Is there in osmand a clear identification of these limitations on the map?

**Reply by vshcherb** [reply]
Nope, it's only routing taken into account
